src/golang/lib/models/operator_result.go

```diff
@@ -3,7 +3,7 @@ package models
 import (
 	"strings"
 
-	"github.com/aqueducthq/aqueduct/lib/models/shared"
+	"github.com/aqueducthq/aqueduct/lib/collections/shared"
```

Comment:
are we still keeping `collections/shared`?

Reply:
We are using collections/shared structs because it is widely use in our codebase and as a result would require our database refactor to take a much longer time since we cannot cast the models/shared types to collection/shared types even though they are duplicates.